src/test/java/core/basesyntax/converter/DataConverterImplTest.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.converter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import core.basesyntax.transaction.FruitTransaction;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class DataConverterImplTest {

```

Comment:
You also have several data validations in this class (raw has 3 parts, quantity validation). So let’s test it ;)